utils/stress-ng/patches/010-dont_define_fortify.patch

```diff
@@ -0,0 +1,13 @@
+--- a/stress-ng.h
++++ b/stress-ng.h
+@@ -34,10 +34,6 @@
+ #ifndef _LARGEFILE64_SOURCE
+ #define _LARGEFILE64_SOURCE
+ #endif
+-#if !defined(__PCC__) && 	\
+-    !defined(__TINYC__)
+-#define _FORTIFY_SOURCE 2
+-#endif
```

Comment:
this patch is fine, but i don't see it fixing errors;
it's fixing warnings;

if possible i would send it upstream to have it discussed with them;
they are reasonably quick;

i would not carry patches in our tree if they don't fix anything more grave;